api/core/model_runtime/model_providers/openrouter/openrouter.yaml

```diff
@@ -0,0 +1,75 @@
+provider: openrouter
+label:
+  en_US: openrouter.ai
+icon_small:
+  en_US: openrouter_square.svg
+icon_large:
+  en_US: openrouter.svg
+background: "#F1EFED"
+help:
+  title:
+    en_US: Get your API key from openrouter.ai
+    zh_Hans: 从 openrouter.ai 获取 API Key
+  url:
+    en_US: https://api.openrouter.ai/
```

Comment:
This URL does not work for me, is it https://openrouter.ai/keys?

Reply:
oh you're right! sorry for that... fixed it in the new commit

Reply:
Emmm, it looks like it's incorrect also
<img width="718" alt="image" src="https://github.com/langgenius/dify/assets/45712896/23c764ba-ee12-4fa2-8ea8-e3d395a5563a">

Reply:
Sorry I added `/key` but didn't remove the subdomain part of the url

Please check now